assemblies/bias-monitoring-tutorial.adoc

```diff
@@ -0,0 +1,36 @@
+:_module-type: ASSEMBLY
+
+ifdef::context[:parent-context: {context}]
+
+:context: bias-tutorial
+
+[id="bias-monitoring-tutorial_{context}"]
+= Bias monitoring tutorial - Gender bias example
+
+[role='_abstract']
+Step-by-step guidance for using TrustyAI in {productname-short} to monitor machine learning models for bias.
+
+include::modules/t-bias-introduction.adoc[leveloffset=+1]
+
+include::modules/t-bias-setting-up-your-environment.adoc[leveloffset=+1]
+
+include::modules/t-bias-deploying-models.adoc[leveloffset=+1]
+
+include::modules/t-bias-sending-training-data-to-the-models.adoc[leveloffset=+1]
+
+include::modules/t-bias-examining-trustyai-model-metadata.adoc[leveloffset=+1]
```

Comment:
based on your response to my comment in the google draft - change the include statement for t-bias-examining-trustyai-model-metadata.adoc to be before t-bias-sending-training-data-to-the-models.adoc

Reply:
Looking at the original tutorial again, I see that examining-trustyai-model-metadata.adoc is really another way to verify the procedure in  t-bias-sending-training-data-to-the-models.adoc, so I moved the info to the Verification section of that module.